libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1173,7 +1175,7 @@ bool NavEKF3::setOriginLLH(const Location &loc)
     if (!core) {
         return false;
     }
-    if (_fusionModeGPS != 3) {
+    if (_sources.getPosXYSource() != AP_NavEKF_Source::SourceXY::EXTNAV) {
```

Comment:
inconsistent conversion of _fusionModeGPS != 3

Reply:
I've changed this to be consistent with the other check so that both are now, "if (_sources.getPosXYSource() == AP_NavEKF_Source::SourceXY::GPS) {"